src/types.ts

```diff
@@ -266,13 +267,18 @@ export interface NodeRendererProps {
 
 export type NodeRenderer = (args: NodeRendererProps) => ReactNode;
 
-export interface ClusterLabelRendererProps {
-  label: string;
-  opacity?: number;
-  fontUrl?: string;
+export interface ClusterRendererProps {
+  offset: number;
+  rad: number;
+  padding: number;
+  opacity: SpringValue<number>;
+  label: {
+    position: SpringValue<[number, number, number]>;
```

Comment:
I don't love how `position` is coupled to `SpringValue` like this. No way to just pass number?

Reply:
I do it to get rid casting to any as it was previously

![Screenshot 2024-11-21 at 17 05 05](https://github.com/user-attachments/assets/4fb4877c-861c-4683-9875-aed06c3f32b5)

Reply:
I can do it
```
position: SpringValue<[number, number, number]> | [number, number, number] ;
```

Reply:
I guess I'm trying to say we shouldn't pass a SpringValue, it will cause a ton of re-renders.

Reply:
But how it will work if animation is enabled?